internal/services/keyvault/key_vault_certificate_contacts_resource.go

```diff
@@ -117,8 +116,15 @@ func (r KeyVaultCertificateContactsResource) Create() sdk.ResourceFunc {
 
 			existing, err := client.GetCertificateContacts(ctx, *keyVaultBaseUri)
 			if err != nil {
+				// If we don't have access to the dataplane due to the public network access
```

Comment:
In this case, shall we just error out as the user's intent is to create the resource, but lacks of permission?